frontend/src/app/login/login.drv.js

```diff
@@ -0,0 +1,24 @@
+/*
+ * Directive for login formular
+ */
+angular.module('flocs.user')
+.directive('loginForm', function(){
+	return{
+		restrict:'EA',
+		template:
```

Comment:
Login page is suposed to have diferent structure than login directive so
maybe different file name will be used.

Could you give me hint how to easily link file template to directive and if
it work same way?


On Mon, Nov 23, 2015 at 3:09 PM, Jiří Mauritz <notifications@github.com>
wrote:

> In frontend/src/app/login/login.drv.js
> <https://github.com/effa/flocs/pull/56#discussion_r45606013>:
>
> > @@ -0,0 +1,24 @@
> > +/*
> > + * Directive for login formular
> > + */
> > +angular.module('flocs.user')
> > +.directive('loginForm', function(){
> > +	return{
> > +		restrict:'EA',
> > +		template:
>
> put the template in a file, most likely into login.tpl.html for this html
> and use reference to the file here
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/effa/flocs/pull/56/files#r45606013>.
>



-- 
Best regards,

Jakub Peschel